### PR TITLE
fix: ask for 3 accounts (signer, pda, system_program) when parsing Sonala inbound signer address

### PR DESCRIPTION
**Pull request by ws4charlie**
# Description

- Backport Athens3 hotfix https://github.com/zeta-chain/node/pull/2778
- Use default relayer key path `~/.zetacored/relayer-keys` if not set in zetaclient config file

# How Has This Been Tested?

- [x] Tested CCTX in localnet
- [ ] Tested in development environment
- [x] Go unit tests
- [ ] Go integration tests
- [ ] Tested via GitHub Actions

## Summary by CodeRabbit

- **New Features**
	- Introduced a new entry in the changelog detailing the updated deposit process for the Solana gateway, requiring three accounts (signer, PDA, system program).
	- Added a new JSON file representing transaction results from the Solana blockchain, enhancing transaction analysis.

- **Improvements**
	- Enhanced error handling for deposit instructions, improving robustness and clarity in execution.
	- Updated configuration to provide a default path for relayer keys, streamlining user setup.

- **Bug Fixes**
	- Adjusted the expected number of accounts for deposit instructions from four to three, simplifying the deposit process. 

- **Tests**
	- Updated test cases to reflect current transaction data and expected outcomes, improving test accuracy.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
 * [2644](https://github.com/zeta-chain/node/pull/2644) - add created_timestamp to cctx status
 * [2673](https://github.com/zeta-chain/node/pull/2673) - add relayer key importer, encryption and decryption
 * [2633](https://github.com/zeta-chain/node/pull/2633) - support for stateful precompiled contracts.
+* [2787](https://github.com/zeta-chain/node/pull/2787) - ask for 3 accounts (signer, pda, system_program) on solana gateway deposit
```
Comment (lumtis) [inline changelog.md:18]: This is rather a fix isn't it?

**Review by lumtis — CHANGES_REQUESTED** [review]
Looks good to me but can we rename the PR and give the actual purpose of it?

This is not a backport, the backport is what we merged on v19

File: changelog.md (hunk 1 above)
Comment (ws4charlie) [reply changelog.md]: https://github.com/zeta-chain/node/pull/2787/commits/7f0b7460338a6bb4db75b98bfddd4a3aadd034a4

**Comment by ws4charlie** [comment]
> Looks good to me but can we rename the PR and give the actual purpose of it?
> 
> This is not a backport, the backport is what we merged on v19

The title of this PR is updated.

**Review by lumtis — APPROVED** [review]

**Pull request merged**